<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>meshesModeQWidget</class>
 <widget class="QWidget" name="meshesModeQWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>74</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="meshesModeQGroup">
     <property name="title">
      <string>Meshes Mode:</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="meshesModeQLabel">
        <property name="text">
         <string>Source Meshes:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="meshesModeQLine">
        <property name="placeholderText">
         <string>*</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="meshesModeQButton">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>&lt;&lt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
